tweaks/Addicted/DrugPump.yml

```diff
@@ -0,0 +1,129 @@
+# Game.AddToInventory("Items.AdvancedDrugPumpCommon")
+# Game.AddToInventory("Items.AdvancedDrugPumpUncommon")
+# Game.AddToInventory("Items.AdvancedDrugPumpUncommonPlus")
+# Game.AddToInventory("Items.AdvancedDrugPumpRare")
+# Game.AddToInventory("Items.AdvancedDrugPumpRarePlus")
+# Game.AddToInventory("Items.AdvancedDrugPumpEpic")
+# Game.AddToInventory("Items.AdvancedDrugPumpEpicPlus")
+# Game.AddToInventory("Items.AdvancedDrugPumpLegendary")
+# Game.AddToInventory("Items.AdvancedDrugPumpLegendaryPlus")
+# Game.AddToInventory("Items.AdvancedDrugPumpLegendaryPlusPlus")
+Items.AdvancedDrugPump$(tier):
+  $instances: &DrugPumpInstances
+    - { tier: Common, upgrade: Uncommon, humanity: 4, consumechance: 90 }
+    - { tier: Uncommon, upgrade: UncommonPlus, humanity: 6, consumechance: 80 }
+    - { tier: UncommonPlus, upgrade: Rare, humanity: 6, consumechance: 80 }
+    - { tier: Rare, upgrade: RarePlus, humanity: 8, consumechance: 70 }
+    - { tier: RarePlus, upgrade: Epic, humanity: 8, consumechance: 70 }
+    - { tier: Epic, upgrade: EpicPlus, humanity: 10, consumechance: 60 }
+    - { tier: EpicPlus, upgrade: Legendary, humanity: 10, consumechance: 60 }
+    - { tier: Legendary, upgrade: LegendaryPlus, humanity: 12, consumechance: 50 }
+    - { tier: LegendaryPlus, upgrade: LegendaryPlusPlus, humanity: 12, consumechance: 50 }
+    - { tier: LegendaryPlusPlus, upgrade: None, humanity: 12, consumechance: 50 }
+  $base: Items.AdvancedBloodPump$(tier)
+  displayName: LocKey#Gameplay-Cyberware-DisplayName-DrugPump
+  localizedDescription: LocKey#Gameplay-Cyberware-LocalizedDescription-DrugPump
+  icon:
+    atlasResourcePath: addicted\gameplay\gui\common\icons\items\DrugPump.inkatlas
+    atlasPartName: cw_circulatory_drugpump
+  appearanceName: None
+  appearanceResourceName: None
+  quality: Quality.$(tier)
+  nextUpgradeItem: Items.AdvancedDrugPump$(upgrade)
+  variants:
+    - Variants.Humanity$(humanity)Cost
+  OnEquip:
+    - Items.AdvancedBloodPump$(tier)_inline0
+    - Items.AdvancedDrugPump$(tier)Logic
+    - Attunements.TechConsumableEffectiveness
+  objectActions:
+    - CyberwareAction.Use$(tier)DrugPump
+
+Items.AdvancedDrugPumpNone: # Fake item to prevent error in log
+  $type: gamedataItem_Record
+  displayName: LocKey#UI-Settings-Language-Debug
```

Comment:
Are we sure that this does not suggest a dummy item as the next upgrade ?
I need to test it :)

Reply:
Just gonna quote you from Discord as a reminder:
> And to answer your question about the dummy item, I used $type, so the game def won't be upgrading to an item that has like no data whatsoever.